cmd/cve/generate.go

```diff
@@ -76,6 +88,63 @@ func main() {
 	createEmptyFiles(*dbDir, "recent")
 }
 
+func downloadLatestRelease(dbDir string, debug bool, logger log.Logger) error {
+	// Download the latest release
+	err := nvd.DownloadCVEFeed(dbDir, "", debug, logger)
+	if err != nil {
+		return fmt.Errorf("download cve feed: %w", err)
+	}
+
+	// gunzip json files
+	files, err := filepath.Glob(filepath.Join(dbDir, "nvdcve-1.1-*.json.gz"))
+	if err != nil {
+		return fmt.Errorf("glob json files: %w", err)
+	}
+	for _, file := range files {
+		err = gunzipFileToDisk(file, dbDir)
+		if err != nil {
+			return fmt.Errorf("gunzip file to disk: %w", err)
+		}
+	}
+
+	// Download the last mod start date
+	err = downloadLatestGitHubAsset(dbDir, "last_mod_start_date.txt")
```

Comment:
Nit: Should `nvd.DownloadCVEFeed` do the download of the `last_mod_start_date.txt` (maybe helpful when debugging customer issues?)

Reply:
downstream it's using the facebookincubator/nvdtools sync, but i think worth it once we pull in that dependency into our repo